civihr_employee_portal/views/templates/views-view-table--Tasks--block.tpl.php

```diff
@@ -40,19 +50,11 @@
 $contacts = array();
 
 foreach ($rows as $row):
-    $rowType = null;
-    if (strip_tags($row['task_contacts_1']) == $civiUser['contact_id']):
-        $rowType = 'task-my';
-    endif;
-    if (strip_tags($row['task_contacts_2']) == $civiUser['contact_id'] && strip_tags($row['task_contacts_1']) != $civiUser['contact_id']):
-        $rowType = 'task-delegated';
-    endif;
-    if (!$rowType):
-        continue;
-    endif;
-    $contactsIds[strip_tags($row['task_contacts'])] = 1;
-    $taskFiltersCount[_get_task_filter_by_date($row['activity_date_time'])]++;
-    $taskFiltersCount[0]++;
+  if (taskAssignedToUser($row, $civiUser['contact_id'])) { continue; }
```

Comment:
Usually we avoid using inline code format with conditions like the one { continue; } should be changed to 
if ( condition ) {
  continue;
}